marconi-core-json-rpc/src/Marconi/Core/JsonRpc.hs

```diff
@@ -0,0 +1,64 @@
+{-# LANGUAGE RankNTypes #-}
+
+{- |
+ This library provides some utilities for running `marconi-core`'s 'Queryable' interface throught a
+ JSON-RPC HTTP server.
+
+ Particularly, these utilities use the `servant-server` Haskell package.
+-}
+module Marconi.Core.JsonRpc where
```

Comment:
Can we use explicit export even if it doesn't sound helpful now? (it sounds like a good practice, especially if we want to tidy the doc later)